src/account/Settings.tsx

```diff
@@ -416,7 +413,7 @@ export class Account extends React.Component<Props, State> {
               title={t('editProfile')}
               onPress={this.goToProfile}
             />
-            {!numberVerified && verificationPossible && (
```

Comment:
Looks like we may want to keep this for when `hideVerification` is true.

Reply:
i think that we resolved this in the conversation above :)